deepdrivemd/agents/stream/dbscan.py

```diff
@@ -0,0 +1,363 @@
+import json
+import random
+from pathlib import Path
+from typing import List, Tuple, Dict, Union
+import numpy as np
+import glob
+import subprocess
+import time
+import sys
+import os
+import gc
+from numba import cuda
+
+from deepdrivemd.utils import Timer, timer, t1Dto2D, parse_args
+from deepdrivemd.data.api import DeepDriveMD_API
+from deepdrivemd.agents.stream.config import OutlierDetectionConfig
+import tensorflow.keras.backend as K
+from  MDAnalysis.analysis.rms import RMSD
+
+import hashlib
+import pickle
+from OutlierDB import *
+from lockfile import LockFile
+from aggregator_reader import *
+
+import cupy as cp
+from cuml import DBSCAN as DBSCAN
+
+# from cuml import TSNE
+from sklearn.manifold import TSNE
+import cuml
+
+from deepdrivemd.models.keras_cvae_stream.model import conv_variational_autoencoder
+
+from dask.distributed import Client, wait
```

Comment:
Removing unused imports (this entire bunch appears to be unused). An IDE should easily pick this up.

Reply:
Done in dbscan and other places.